src/test/groovy/com/bmuschko/gradle/docker/utils/TaskStateHelperTest.groovy

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.bmuschko.gradle.docker.utils
+
+import org.gradle.api.Project
+import spock.lang.Specification
+
+class TaskStateHelperTest extends Specification {
```

Comment:
I'd probably move this into the integration test SourceSet and use `ProjectBuilder` instead.